Library/Formula/libxmp-lite.rb

```diff
@@ -0,0 +1,10 @@
+class LibxmpLite < Formula
```

Comment:
If the library name has changed that seems a good reason to boneyard/new formula, yeh.